guess_a_number_version_1.py

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env python
+
+from random import randint
+
+def main():
```

Comment:
I created a main function. Don't be like me, though: don't put an empty line at the start of a function.